.github/workflows/ci.yml

```diff
@@ -0,0 +1,54 @@
+# Github worflows for Ocean
+
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  main:
+    name: Run
+    strategy:
+      # Disable `fail-fast` because we want the whole test suite to run even if one
+      # of the nigthly is broken
```

Comment:
typo: should be `nightly` :-)

Reply:
BTW I assume here you are talking about nightly compiler builds (which will presumably be added later as they are not in the current setup)?  It might be worth clarifying that in the comment.

Also, it's not just about nightlies!  I've experienced failures with more recent DMD releases that passed with earlier ones (usually because of a deprecation-related issue), or failures with prod that did not show up with devel.  So it's useful to see all the matrix builds even if no nightlies are involved.

Maybe rephase:

```
Disable `fail-fast` because even if things fail for one compiler release
or build flavour we still want to see how things perform with the others
```

Reply:
Good point, will do.